Apps/APN-UIKit/APN UIKit/View/BaseViewController.swift

```diff
@@ -16,4 +16,8 @@ class BaseViewController: UIViewController {
         let versionDetail = "SDK: Customer.io iOS SDK \(SdkVersion.version)\nApp: Ami app \(appVersion ?? "")"
         return versionDetail
     }
+    func setAccessibilityIdentifierTo(_ element: UIView, value: String) {
+        element.isAccessibilityElement = true
```

Comment:
By default, the `isAccessibilityElement` property is set to `false`. To utilize the accessibility element, we need to set it to `true`.

Since `UIBarButtonItem` is not of type `UIView`, we cannot directly pass `backButton` as a parameter.

Thank you for sharing the idea with me, but I still prefer keeping the name as it is. The reason is that it sets the accessibility identifier. The parent functions that call `setAccessibilityIdentifierTo` aim to provide clarity by indicating that the function is used for setting accessibility identifiers for Appium. The function is named `addAccessibilityIdentifiersForAppium`. I hope this adds more clarity.